embed_grader/serapis/templates/serapis/testbed_status_list.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'serapis/base_sidebar.html' %}
+{% load staticfiles %}
+
+{% block profile-usertitle-name %}
+  Testbed
+{% endblock %}
+
+{% block profile-usertitle-job %}
+  <br/>
+{% endblock %}
+
+{% block testbed-status-list-active %}active{% endblock %}
+
+{% block main-content %}
+  <div class="main-content">
+    <h1 class="page-header">Testbed Status List</h1>
+
+    {% if not testbed_list %}
+      <p>No testbeds available to view.</p>
+    {% else %}
+      <div class="table-responsive">
+      <table class="table table-hover">
```

Comment:
Please double check the indentation. For example, the <table> should have 2 more blanks than the previous line. Please note we also indent the codes embraced in template tags, for example, line 32 to line 39 should be indented